coderd/aitasks.go

```diff
@@ -0,0 +1,68 @@
+package coderd
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/v2/coderd/httpapi"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+// @Summary Get AI tasks prompts
+// @ID get-ai-tasks-prompts
+// @Security CoderSessionToken
+// @Produce json
+// @Tags AITasks
+// @Param build_ids query string true "Comma-separated workspace build IDs"
```

Comment:
```suggestion
// @Param build_ids query string true "Comma-separated workspace build IDs" format(uuid)
```

See https://github.com/coder/coder/blob/f1269312198cbee8f5a4a996a5bdec5ccf0dcaff/coderd/workspacebuilds.go#L109